contrib/helm/clair/values.yaml

```diff
@@ -5,9 +5,9 @@ replicaCount: 1
 logLevel: info
 insecureTls: false
 image:
-  repository: quay.io/coreos/clair-git
-  tag: latest
-  pullPolicy: Always
+  repository: quay.io/coreos/clair
+  tag: v2.0.1
```

Comment:
Because this code is in master, it should be tied to deploying master. When we cut a stable release, we should tie this to a particular version.